build-tools/packages/build-tools/src/type-test-generator/compatibility.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
Please add a note why namespaces are ok in this file, since we discourage them in general.

Reply:
Done.  (The answer is that they are non-exported namespaces to help organize/delimit inline compile-time tests.)